### DISCUSSION TITLE
Enable to change currency format from dollars to euros

### DISCUSSION THREAD

**Discussion by zsellami** [apache/superset#18408] [Ideas]
Hello,
I want to show euros values on dashboard. But only the $ format are enable. When i changed the $ symbol to € i obtained an error.
How can I show euros value in a chart ?

I want something like "y_axis_format": "$,.2f" ==> to "y_axis_format": ",.2f€"

Regards,
Zied Sellami

[a comment by an automated account is not shown]

**Comment by junlincc** [comment]
Thank you, Zied for suggesting the enhancement. This feature is not currently on Superset 1.0 roadmap so we won't be able to get to it till early next year likely. However, this looks like a 'good first issue' for any community contributors to work on. If you are interested in contributing, I can point you to the right direction and connect you with you engineers.

**Comment by dstadz** [comment]
Hey @junlincc, I am interested in contributing to this issue. Is it still available to be worked on?

**Comment by willbarrett** [comment]
Yup, go ahead and work on it!

**Comment by leocape** [comment]
Also with specifying a currency it would be nice to have the option to remove decimals eg. `$,.2f` becomes  `$,.0f`

**Comment by carlosaccp** [comment]
Is this issue still open? I would like to help!

**Comment by sahilkamboj3** [comment]
Is the issue still open? I would love to join and help out

**Comment by nytai** [comment]
@sahilkamboj3 I believe it is. If you open a PR, I can ensure it gets reviewed.

**Comment by junlincc** [comment]
@carlosaccp, @sahilkamboj3 @dstadz sorry I missed your comments!!  If you all are still interested in contributing, feel free to reach out to me in Slack. you will need to add this y-axis format to multiple viz plug-ins. We can help you navigate if needed. thanks!

**Comment by juanmantelli** [comment]
Any news about this feature?

**Comment by JavierLopezT** [comment]
Very necessary feature

**Comment by crmne** [comment]
absolutely essential

**Comment by lf-floriandin** [comment]
Also a huge fan.
I am pretty much a fan of every feature that improves the internationalization of Superset. Seems very America-focused so far.

**Comment by yousoph** [comment]
https://github.com/apache/superset/pull/24718 and https://github.com/apache/superset/pull/24517 add support for more currency symbols

**Comment by gferrette** [comment]
Hello!

Please, is it possible to add new currency symbols and new d3 format on file values.yaml when using kubernetes helm chart?

Thanks in advance!
